ui/src/message_popup/author/question_page.jsx

```diff
@@ -104,8 +106,11 @@ export default React.createClass({
     console.log("Saving the following new question:");
     const studentIds = this.state.students.map(student => student.id);
     const text = this.state.questionText;
+    const {originalQuestion} = this.props;
     const {goodExamples, badExamples} = this.getExamples();
-    const id = _.maxBy(allQuestions, question => question.id).id + 1;
```

Comment:
I think adding the higher level component above this which would also handle the question loading instead of `NewQuestionPage` and `EditQuestionPage` would work.